src/boringssl/gen_build_yaml.py

```diff
@@ -164,5 +164,10 @@ def WriteFiles(self, files, asm_outputs):
 
   print yaml.dump(g.yaml)
 
+except OSError as err:
+  type, value, traceback = sys.exc_info()
+  raise RuntimeError, (
```

Comment:
Instead of creating a new exception, you could recycle the original one after attaching your message. For example:

```
except OSError as err:
  if not err.args:
      err.args=('',)
  err.args = err.args + 'Y U NO GO IN PATH' 
  raise
```
`raise` on its own will re-throw the caught exception.